homeassistant/components/tuya/sensor.py

```diff
@@ -1168,6 +1168,15 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             state_class=SensorStateClass.MEASUREMENT,
         ),
     ),
+    # Pet Water Feeder
+    # https://developer.tuya.com/en/docs/iot/f?id=K9gf46aewxem5
+    "cwysj": (
+        TuyaSensorEntityDescription(
+            key=DPCode.WATER_LEVEL,
+            translation_key="water_level",
```

Comment:
I added test for FAULT sensor, i think u can reuse it. But it is not merged for now https://github.com/home-assistant/core/pull/129437